Integrations/GoogleCloudStorage/GoogleCloudStorage.yml

```diff
@@ -115,105 +115,105 @@ script:
       required: true
       secret: false
     deprecated: false
-    description: Permanently deletes a bucket
+    description: Deletes a bucket.
     execution: false
     name: gcs-delete-bucket
   - arguments:
     - default: true
-      description: Name of the bucket to list objects from
+      description: Name of the bucket in which to list objects.
       isArray: false
       name: bucket_name
       required: true
       secret: false
     deprecated: false
-    description: Retrieves the list of objects in a bucket
+    description: Retrieves the list of objects in a bucket.
     execution: false
     name: gcs-list-bucket-objects
     outputs:
     - contextPath: GCS.BucketObject.Name
-      description: Object name
+      description: Object name.
       type: String
     - contextPath: GCS.BucketObject.Bucket
-      description: Name of the bucket containing this object
+      description: Name of the bucket containing the object.
       type: String
     - contextPath: GCS.BucketObject.ContentType
-      description: Content-Type of the object data
+      description: Content-Type of the object data.
       type: String
     - contextPath: GCS.BucketObject.TimeCreated
-      description: Object creation time
+      description: Object creation time.
       type: Date
     - contextPath: GCS.BucketObject.TimeUpdated
-      description: Last object modification time
+      description: Last time object was modified.
       type: Date
     - contextPath: GCS.BucketObject.TimeDeleted
-      description: Object deletion time (available if the object is archived)
+      description: Object deletion time (available if the object is archived).
       type: Date
     - contextPath: GCS.BucketObject.Size
-      description: Object size in bytes
+      description: Number of objects in bytes.
```

Comment:
@richardbluestone, this field is the size of the object in bytes, not the number of objects.

Reply:
@panro, I've updated the description to Object size in bytes.